src/game_interpreter.cpp

```diff
@@ -812,6 +812,8 @@ bool Game_Interpreter::ExecuteCommand(lcf::rpg::EventCommand const& com) {
 			return CmdSetup<&Game_Interpreter::CommandManiacGetGameInfo, 8>(com);
 		case Cmd::EasyRpg_SetInterpreterFlag:
 			return CmdSetup<&Game_Interpreter::CommandEasyRpgSetInterpreterFlag, 2>(com);
+		case static_cast<Cmd>(3032): // Maniac_Zoom
```

Comment:
Should open a PR to liblcf to add this command to the enum: https://github.com/EasyRPG/liblcf/blob/master/generator/csv/enums_easyrpg.csv#L14

Reply:
Ah my bad, if there is more than one missing, I would say don't worry about it as a blocker on this PR, but someone should clean up the missing ones